test/built-ins/Set/prototype/difference/set-like-array.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2023 Kevin Gibbons, Anthony Frehner and Kevin Gibbons. All rights reserved.
```

Comment:
I appreciate @bakkot as much as the next guy, but isn't this a bit over the top?

Reply:
Lol whoops, messed up my regex.